### PR TITLE
Add additional style overrides to Product Drawer. SITES-5126

### PR DESCRIPTION
**Pull request by tobs1701**
Also, hide toggle text at smaller viewport width.

Addresses https://github.com/Faithlife/styled-ui/issues/280.

### CODE REVIEW

File: components/product-drawer/styled.jsx [hunk 1]
```diff
@@ -36,8 +36,11 @@ export const ProductDrawerToggle = styled.button`
 export const ProductDrawerToggleText = styled.span`
 	display: none;
 
-	@media (min-width: ${mediaSizes.tablet}) {
+	@media (min-width: ${mediaSizes.phone}) {
 		display: inline;
 		margin-left: 8px;
+		font-family: ${({ styleOverrides }) => styleOverrides.toggleTextFontFamily || 'inherit'};
+		font-size: ${({ styleOverrides }) => styleOverrides.toggleTextFontSize || 'inherit'};
+		color: ${({ styleOverrides }) => styleOverrides.toggleTextColor || 'inherit'};
```
Comment (bryanrsmith) [inline components/product-drawer/styled.jsx]: You only want to apply the style overrides at larger viewports? Why? That seems surprising.

Comment (tobs1701) [reply components/product-drawer/styled.jsx]: Smaller viewports are `display: none`.

Comment (bryanrsmith) [reply components/product-drawer/styled.jsx]: That's a decision that could easily change. I'd suggestion limiting the scope of the media queries to the properties that actually need to change. A style like color wouldn't likely ever be affected by the viewport width, so putting it in a media query makes the code harder to understand.

Comment (tobs1701) [reply components/product-drawer/styled.jsx]: https://github.com/Faithlife/styled-ui/pull/281/commits/05eee6274ce124a3485f5408761ed0dc6f53f5dd

**Pull request merged**